src/test/Testcafe/page-objects/designerPage.js

```diff
@@ -79,6 +79,9 @@ export default class DesignerPage {
     this.deploys = this.deployTable.find('tr');
     this.deployConfirm = Selector("#deployPopover");
     this.deployStatus = Selector('p').withText('deployer versjon');
+    this.noDeployAccessText = "Du har ikke rettigheter til å starte en deploy til ";
+    this.noDeployAccess = (environment == 'prod') ? Selector('p').withText(this.noDeployAccessText + "PRODUCTION-miljøet") :
+      Selector('p').withText(this.noDeployAccessText + "AT22-miljøet");
```

Comment:
is there no need to make this more generic to support all environments or is this the only environment the test user can't deploy to ?

Reply:
The test user does not have write access to the repo or no access to the teams in org repo. this test is just to verify that the users not in team cannot deploy. it is unnecessary to test the same for every environment , so deploys are AT22 are tested in dev studio and prod in prod studio.